copier.yml

```diff
@@ -62,8 +62,29 @@ component_owner:
     type: str
     help: |
         Owner of the component in the Developer Portal Software Catalogue.
-        This will normally be group:default/<your_group>
-    default: "group:default/sscc"
+        This will normally be group:default/<your group>
+
+component_type:
+    type: str
+    help: |
+        Type of the component in the Developer Portal Software Catalogue.
+        Most likely service, library or application (without quotes).
+
+component_lifecycle:
+    type: str
+    help: |
+        Project's current lifecycle stage, to be displayed in the Developer 
+        Portal Software Catalogue.
+    choices:
+        - alpha
+        - beta
+        - deprecated
+        - development
+        - experimental
+        - production
+        - test
+        - unknown
```

Comment:
Could we use the [common lifecycle stages](https://dev-portal.diamond.ac.uk/guide/developer-portal-catalog/docs/references/common-lifecycle-stages/) from the developer portal?